src/components/forms/Form/Form.tsx

```diff
@@ -6,12 +6,13 @@ interface FormProps {
   onSubmit: (event: React.FormEvent<HTMLFormElement>) => void
   className?: string
   large?: boolean
+  role?: string
 }
 
 export const Form = (
   props: FormProps & React.FormHTMLAttributes<HTMLFormElement>
 ): React.ReactElement => {
-  const { onSubmit, children, className, large, ...formProps } = props
+  const { onSubmit, children, className, large, role, ...formProps } = props
```

Comment:
No need for changes in this file.  The `role` is included in `...formProps` (specifically `role` is in the `HTMLAttributes` which is accepted as props).